app/models/event.rb

```diff
@@ -14,4 +14,8 @@ class Event < ApplicationRecord
   scope :requiring_notification, -> do
     where('date = ?', Date.current - NOTIFICATION_PERIOD)
   end
+
+  def duration
+    presentations.map(&:duration).sum
```

Comment:
Alright so we actually can't do this on the DB level because there are instances in which we will want to call this method before presentations have been saved to the database and are hanging around in "memory?" Not sure if that's the right term but the map is a better solution here.